fts-cron/Dockerfile_cpp

```diff
@@ -33,6 +35,8 @@ RUN yum -y install wget && \
     yum clean all && \
     rm -rf /var/cache/yum
 
+CMD ["/bin/bash", "-c", "while true; do sleep 30; done;"]
```

Comment:
What is this needed for?

Reply:
And I would put it down next to `ENTRYPOINT`